tools/fuzzer/fuzzer_test.cc

```diff
@@ -156,6 +156,13 @@ class FakeGeneratorRng : public GeneratorRng {
     return fuzzer::Field(std::move(type), std::move(field_name));
   }
 
+  const fuzzer::Function& pick_function(
+      const std::vector<std::reference_wrapper<const fuzzer::Function>>&)
+      override {
+    assert(false && "Not implemented yet!");
+    return fuzzer::Function("dummy", {});
```

Comment:
Returning reference to a temporary value. Either create this dummy function at a global/class scope, or just return by value. I think for our purposes copying `Function` object is not that bad.

Reply:
Replaced returning by reference with returning by value.